liqd_product/assets/scss/components/_cms.scss

```diff
@@ -13,12 +12,30 @@
     font-weight: 600;
 }
 
+.homepage-title {
+    display: inline-block;
+    border-bottom: solid 8px $platform-brand-primary;
+    padding-bottom: 23px;
+}
+
+.homepage-title--large {
+    @media screen and (max-width: $breakpoint) {
```

Comment:
With the reorganising the heading we would also not ask for min and max and use that one pixel inbetween?

Reply:
what do you mean by pixel in between?